data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/BeaconRestApi.java

```diff
@@ -161,19 +166,32 @@ private String readResource(final String fileName, Charset charset) throws IOExc
   }
 
   private void addExceptionHandlers() {
+    app.exception(ChainDataUnavailableException.class, (e, ctx) -> ctx.status(SC_NO_CONTENT));
     app.exception(
-        ChainDataUnavailableException.class,
+        NodeSyncingException.class,
         (e, ctx) -> {
-          ctx.status(SC_NO_CONTENT);
+          ctx.status(SC_SERVICE_UNAVAILABLE);
+          setErrorBody(ctx, () -> BadRequest.serviceUnavailable(jsonProvider));
```

Comment:
this breaks a lot of the old api's if we care about that?

Reply:
The diff makes it look a bit confusing.  `ChainDataUnavailableException` still returns `SC_NO_CONTENT` as always.  `NodeSyncingException` previously return 500 internal server error but now returns `SC_SERVICE_UNAVAILABLE`.

Reply:
ah ok